mantidproject/mantid
# Fix crystallography convention for LoadMD

Description of work:
Only change first 3 dimensions in LoadMD when file is loaded with the opposite convention.

Change: Assume the Q dimensions are those that have names starting with `[` such as `[H,0.5H,0]`, or `Q_` such as `Q_sample_x`. The change in sign should apply only to those

**To test:**
https://www.dropbox.com/s/12uaprzg8w8ua1o/TOPAZ_20575_VT_HKL_Cubic_Cell.nxs?dl=0
LoadMD 4D file that was saved in Crystallography Convention from Dropbox and use Crystallography convention unchecked during load.  Check in SliceViewer that temperature is positive.

Fixes #17518

_Does not need to be in the release notes._

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
